app/controllers/api/v1/Web.java

```diff
@@ -1435,4 +1474,15 @@ public int compare(JsonObject a, JsonObject b) {
     }
     return sortedJsonArray;
   }
+
+  private static List<Severity> getSortedSeverityKeys(Set<Severity> severities) {
```

Comment:
Does this sort on descending order or ascending order. It might be better to put that as comment